include/dds/core/Reference.hpp

```diff
@@ -244,7 +280,10 @@ class Reference
     Reference& operator=(
             const null_type)
     {
-        throw "Not implemented";
+        DELEGATE_REF_T tmp;
```

Comment:
i remember having compiler problems (caused by another reason) and tried this to make it compile (needed to go forward since i was looking for an error in particular). must have forgotten to place it back as it was/should have been.